src/test/integration/reload-runner/index.js

```diff
@@ -0,0 +1,72 @@
+const WebSocket = require('ws');
+const schema = require('enigma.js/schemas/12.20.0.json');
+const enigma = require('enigma.js');
+
+const host = process.argv.slice(2)[0] || 'localhost';
+const port = host === 'localhost' ? 19076 : 9076;
+
+// create a new session:
+const session = enigma.create({
+  schema,
+  url: `ws://${host}:${port}/app/engineData`,
+  createSocket(url) {
+    return new WebSocket(url);
+  },
+});
+
+let loadData = async function(){
+    const appId = 'reloadapp.qvf';
+    let startTime = Date.now();
+    let global = await session.open();
+    let app;
+
+    try{
+        let appInfo = await global.createApp(appId);
+        app = await global.openDoc(appInfo.qAppId);
+    }
+    catch(e){
+        app = await global.openDoc(appId);
```

Comment:
Why would the first openDoc fail or is it if the app exists?

Reply:
createApp will fail if the app already exists